history.txt

```diff
@@ -1,155 +1,74 @@
-Changes include:
-
-- bumped Bio-Formats version to 5.3.3. This includes several bug fixes.
-  See `Bio-Formats 5.3.3 release history <https://www.openmicroscopy.org/site/support/bio-formats5.3/about/whats-new.html>`_
-- work on the Web API
-- removal pdf doc generation
-- improvement to how colors are assigned to channels
-- disabled jquery cache
-- support for projection along T
-- fixed numerous tests
-- bug fixes in OMERO.web
-
-
-5.3.0-m7 (January 2017)
------------------------
-
-Developer preview release - **only intended as a developer preview for
-updating code before the full public release of 5.3.0. Use at your own risk**.
-
-Changes include:
-
-- bumped Bio-Formats version to 5.3.2. This includes several bug fixes.
-  See `Bio-Formats release history <https://www.openmicroscopy.org/site/support/bio-formats5.3/about/whats-new.html>`_
-- work on the Web API
-- handled shapes color in clients according to model specification
-- improved open_with option in web
-- added histogram support in clients and server
-- added integration of all official scripts
-- improved SPW grid view in web
-- bug fixes
-
-5.3.0-m6 (November 2016)
-------------------------
-
-Developer preview release - **only intended as a developer preview for
-updating code before the full public release of 5.3.0. Use at your own risk**.
-
-Changes include:
-
-- bumped Bio-Formats version to 5.3.0-m2. This includes JPEG-XR support for CZI.
-  See the `Bio-Formats version history <https://www.openmicroscopy.org/site/support/bio-formats5.3/about/whats-new.html>`_ for more information.
-- fixed numpy-pillow version incompatibility
-- made python testing package public so it can be used by external clients
-
-5.3.0-m5 (November 2016)
-------------------------
-
-Developer preview release - **only intended as a developer preview for
-updating code before the full public release of 5.3.0. Use at your own risk**.
+5.3.0 (March 2017)
+------------------
 
-Changes include:
+A full, production-ready release of OMERO 5.3.0; featuring a major reworking
+of OMERO.web and web apps; dropping support for Windows for the server and for
+deploying OMERO.web using Apache; and introducing new user features and many
+fixes and performance improvements:
+
+-  improved support for many file formats via Bio-Formats 5.4.0
+-  introduced ROI Folders
+-  new UI for displaying Screen Plate Well data in OMERO.web and OMERO.insight
+-  support for lookup tables and reverse intensity rendering
+-  color mapping for multiple channels without set colors has been improved to
+   use RGBRGB rather than RGBBB (i.e. to loop through red, green, blue rather
+   than setting all later channels to blue)
+-  support for histograms in the clients and server
+-  ability to filter by ratings in OMERO.web
+-  added 'Open With...' functionality to OMERO.web
+-  color of shapes is now handled according to the data model, using RGBA
+   rather than ARGB format (an sql script is available to upgrade existing
+   shapes)
```

Comment:
maybe a note 
indicating ``since it is not updated as part of the general upgrade``